src/travel-aid/TravelAidScreenComponent.tsx

```diff
@@ -123,30 +126,24 @@ const TravelAidSection = ({
     fromQuayId,
   );
 
-  const selectedEstimatedCall = serviceJourney.estimatedCalls.find(
-    (e) => e.quay.id === fromQuayId,
+  const situationsForFocused = focusedEstimatedCall.situations ?? [];
+  const noticesForFocused = getNoticesForServiceJourney(
+    serviceJourney,
+    focusedEstimatedCall.quay.id,
   );
-  const situationsForSelected =
-    selectedEstimatedCall?.situations.sort((n1, n2) =>
-      n1.id.localeCompare(n2.id),
-    ) ?? [];
-  const noticesForSelected = selectedEstimatedCall?.notices ?? [];
-  const situationsForFocusedStop =
-    focusedEstimatedCall.situations.filter((situation) =>
-      situationsForSelected?.find((s) => s.id !== situation.id),
-    ) ?? [];
 
   useTravelAidAnnouncements(
     {status, focusedEstimatedCall},
-    situationsForFocusedStop,
+    situationsForFocused,
+    noticesForFocused,
```

Comment:
What did we decide in regards to notices? Will they change for different calls, or can we remove them from the announcements?

Reply:
I thought that there would be another PR for announcements, handling both `notices` and `situations`, and as such didn't really look into it on this PR.

Reply:
Yes, that is the plan to handle it on another PR, but we decided to announce it all once and then filter the new ones based on some kind of announced IDs.

Reply:
If it should be handled in another PR, then I think you should remove the changes to useTravelAidAnnouncements from this PR.

Reply:
I added it to have some base structure before starting the other and also to at least announce the notices as well.